application/src/main/java/org/thingsboard/server/service/subscription/TbAttributeSubscription.java

```diff
@@ -43,12 +43,8 @@ public TbAttributeSubscription(String serviceId, String sessionId, int subscript
     }
 
     @Override
-    public boolean equals(Object o) {
-        return super.equals(o);
+    protected boolean canEqual(final Object other) {
```

Comment:
Why do you change this?

Reply:
Remove all code that is not related to the scope of changes. Create new PR and cover this with TESTS!!!